api/app/models/session.rb

```diff
@@ -0,0 +1,29 @@
+class Session < ApplicationRecord
+  # Associations
+  has_many :attendances, dependent: :destroy
+  has_many :attendees, through: :attendances, source: :user
+  belongs_to :group
+
+  # Validations
+  validates :name, :description, :location, :start_time, :end_time, presence: true
+  validate :end_time_after_start_time
+
+  # After
+  after_create :create_attendances_for_group_members
+
+  private
+
+  def end_time_after_start_time
+    return if end_time.blank? || start_time.blank?
```

Comment:
Esto no es redundante? Porque ya corre la validación de `presence: true` para estos atributos antes..

Reply:
Si lo saco hay que meter un poco de refactoren los test, por ejemplo shoulda-matchers inicializa los atributos en nil para probar la de presencia entonces arrancan a tirar: undefined method `<' for nil:NilClass en varios test.

La verdad que los workaround que vi me parecieron más costosos que dejar esa linea extra en la validación, que te parece?

Reply:
Ahh mirá, puede ser si que pateé por eso. Tranqui, vamos con esta opción y cualquier cosa luego vemos de refactorearlo si es necesario 👍